### DISCUSSION TITLE
there a code bug?

### DISCUSSION THREAD

**Discussion by lixiaoyong12** [apache/airflow#14499] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/34184204/109377304-6b7bb980-7905-11eb-91f9-6aa890216aac.png)

Load_op_links that Initialize the DagBag in line 736 jobs/scheduler_job.py not exsit

**Comment by lixiaoyong12** [comment, marked as answer]
Load_op_links that Initialize the DagBag in line 736 jobs/scheduler_job.py not exsit in DagBag

**Reply by turbaszek** [reply]
It exists in the constructor as far as I see:
https://github.com/apache/airflow/blob/3eceb26672d310523201168d06e7bcc7f59522f7/airflow/models/dagbag.py#L98

**Reply by lixiaoyong12** [reply]
![image](https://user-images.githubusercontent.com/34184204/109383821-084d4f80-7924-11eb-884c-a812ea6c2322.png)
The log should not be an error.

**Reply by turbaszek** [reply]
@lixiaoyong12 I'm not sure if I understand your problem. If you think there's a bug in the code base, please report it using bug report:
https://github.com/apache/airflow/issues/new/choose

**Reply by lixiaoyong12** [reply]
I don't think the log【 Execution date is in future:】 level should be error,  which affects problem determination and Can  change it to Info?
![image](https://user-images.githubusercontent.com/34184204/109384364-33856e00-7927-11eb-876c-bd0151c74549.png)

**Reply by lixiaoyong12** [reply]
![image](https://user-images.githubusercontent.com/34184204/109384434-b3133d00-7927-11eb-8552-74475bab2ec2.png)

**Reply by turbaszek** [reply]
Hm, I think that changing `error` to `warning` may have sense. Would you like to open a PR?

**Reply by lixiaoyong12** [reply]
Yes, I do.
